components/layout/style_ext.rs

```diff
@@ -522,15 +523,20 @@ impl ComputedValuesExt for ComputedValues {
         !self.is_inline_box(fragment_flags)
     }
 
-    /// Returns true if this style has a transform, or perspective property set and
+    /// Returns true if this style has a transform or perspective property set.
+    fn has_transform_or_perspective_style(&self) -> bool {
+        !self.get_box().transform.0.is_empty() ||
+            self.get_box().scale != GenericScale::None ||
+            self.get_box().rotate != GenericRotate::None ||
+            self.get_box().translate != GenericTranslate::None ||
+            self.get_box().perspective != Perspective::None
+    }
+
+    /// Returns true if this style has a transform or perspective property set, and
     /// it applies to this element.
+    #[inline]
     fn has_transform_or_perspective(&self, fragment_flags: FragmentFlags) -> bool {
```

Comment:
How about `has_transform_or_perspective_and_is_transformable`?

Reply:
A bit long, maybe `has_effective_transform_or_perspective`?

Reply:
Makes sense to me. Thanks!